app/models/admin_user.rb

```diff
@@ -17,7 +17,7 @@ def self.root_user
   protected
 
   def not_adding_root_user
-    is_root_user = AdminUser.root_user.id == id
+    is_root_user = AdminUser.root_user&.id == id
```

Comment:
What's the benefit of adding `&`?

Reply:
in case there is no AdminUser in the first place